### ISSUE TITLE
dm_env: Broken Link

### ISSUE THREAD

**Issue by filangel** [deepmind/bsuite#1]
The link to [**dm_env**](https://github.com/deepmind/bsuite/blob/master/bsuite/experiments/github.com/deepmind/dm_env) in [`README.md`](https://github.com/deepmind/bsuite/blob/master/bsuite/experiments/README.md) is broken.

**Comment by yotam** [comment]
Strange, it works for me. It links to https://github.com/deepmind/dm_env.  The raw markdown being

```
Our environments implement the Python interface defined in
[`dm_env`](https://github.com/deepmind/dm_env).
```

**Comment by yotam** [comment]
Ah I see, in `experiments/README.md`. Thanks, I'll fix it.

**Comment by yotam** [comment]
Fixed in a4bfe48
